crates/wasi-http/src/types.rs

```diff
@@ -65,6 +67,14 @@ pub trait WasiHttpView: Send {
     {
         default_send_request(self, request)
     }
+
+    fn is_forbidden_request_header(&mut self, name: &HeaderName) -> bool {
+        default_is_forbidden_request_header(name)
+    }
+
+    fn is_forbidden_response_header(&mut self, name: &HeaderName) -> bool {
+        default_is_forbidden_response_header(name)
+    }
```

Comment:
Would it be useful to add the `HeaderMap` that's being added to as an argument here as well?